snapshot/snapshot.go

```diff
@@ -150,6 +160,8 @@ type snapshotter struct {
 	userxattr                   bool  // whether to enable "userxattr" mount option
 	minLayerSize                int64 // minimum layer size for remote mounting
 	allowInvalidMountsOnRestart bool
+	allowIDMap                  bool
```

Comment:
whats the specific need for this flag? i am not very clear on this.

Reply:
Removed this